samples/nrf9160/aws_fota/README.rst

```diff
@@ -1,17 +1,95 @@
-When it receives notification that an update job is available, it retrieves metadata over MQTT, then retrieves the payload over HTTP, and reports back the status when it completes.
+The sample connects to the configured `AWS MQTT broker`_ and subscribes to several topics related to AWS IoT jobs.
+When an update job is created on the AWS IoT service, the sample receives a notification through MQTT.
+
+Triggered by the notification, the sample retrieves the metadata for the update job over MQTT.
+This metadata contains the location of the new firmware image (which is generated when building the sample, but you must upload it to a server).
+The sample then retrieves the firmware image over HTTP and replaces the current firmware with the downloaded firmware.
+
+See :ref:`lib_aws_fota` for information about the download procedure.
+The `AWS IoT Developer Guide`_ contains all required information about the Amazon Web Services IoT service.
+
+Creating a thing in AWS IoT
+===========================
+
+Before you can run this sample, you must create a *thing* for your board in AWS IoT so that AWS knows about your board.
+This thing must be connected to a security policy.
+For testing, you can use a permissive policy, but make sure to update the policy to be more restrictive before you go into production.
+See `AWS IoT Developer Guide: Basic Policy Variables`_ and `AWS IoT Developer Guide: Security Best Practices`_ for more information about policies.
+
+To create a thing for your board:
+
+1. Log on to the `AWS IoT console`_.
+#. Go to **Secure** > **Policies** and select **Create a policy**.
+#. Enter a name and define your policy.
+   For testing purposes, you can use the following policy (switch to **Advanced mode** to copy and paste it):
+
+   .. code-block:: javascript
+
+      {
+         "Version": "2012-10-17",
+         "Statement": [
+             {
+               "Effect": "Allow",
+               "Action": "iot:*",
+               "Resource": "*"
+             }
+          ]
+       }
+#. Go to **Manage** > **Things** and select **Register a thing** or **Create** (depending on whether you already have a thing registered).
+#. Select **Create a single thing**.
+#. Enter a name.
+   The default name used by the sample is ``nrf-IMEI``, where *IMEI* is the IMEI number of your board.
+   If you choose a different name, make sure to :ref:`configure a custom client ID <configuring>` in the sample before you build it.
+#. Accept the defaults and continue to the next step.
+#. Select **Create certificate** to generate new certificates.
+   Alternatively, you can use existing certificates.
+   In this case, follow the instructions in AWS IoT.
+#. Download the certificates for later use.
+   You need the thing certificate (``*.cert.pem``), the private key (``*.private.key``), and the root CA (choose the Amazon Root CA 1).
+   (?the actual file names have different endings than stated on the site, which do we want to give?)
```

Comment:
I would suggest trying to mirror the site as closely as possible `*-certificate.pem.crt` and `*-private.pem.key` `AmazonRoot*.pem`